es/13_1_Escribiendo_Puzzle_Scripts.md

```diff
@@ -0,0 +1,523 @@
+# 13.1: Escribir guiones de rompecabezas
+
+Los scripts de Bitcoin realmente _no_ tienen que depender del conocimiento de una clave secreta. En cambio, pueden ser rompecabezas de cualquier tipo.
+
+## Escribir guiones de álgebra simples
+
+Nuestro primer script real, de [§9.2: Ejecución de un script de Bitcoin](09_2_Ejecutando_un_Script_Bitcoin.md) fue un acertijo alegbraico. Ese script de Bitcoin, `OP_ADD 99 OP_EQUAL`, podría haberse descrito alternativamente como` x + y = 99`.
+
+Este tipo de script no tiene mucha aplicabilidad en el mundo real, ya que es demasiado fácil reclamar los fondos. Pero, un concurso de resolución de acertijos que distribuya polvo de Bitcoin podría ofrecerlo como un entretenimiento divertido.
```

Comment:
- `Bitcoin dust` can be translated as `pequeñas cantidades de Bitcoin` instead of `polvo de Bitcoin`.

Reply:
This is tricky,  "pequeñas cantidad de bitcoin" pueden ser 0.0001 and not to be dust.  Agree?

Reply:
Totally. What do you think of `pequeñas cantidades de satoshis`, is it better?

Reply:
Is better!